### PR TITLE
Feature/sms message alert error handle

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [X] JavaScript
- [ ] CSS
- [X] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [X] Twilio API
- [X] Refactor/Fix Bugs

## Please describe the change made
All user flow SMS messages have been added and tested. 
Props passing for contact to CurrentTrip and AddTime have been added for SMS message calls 

## Please give a short description of the changes that were made below

- Fix:

- Feature:
TripCompleteMessage added to AddTime => end trip button 
TripCompleteMessage added to CurrentTrip, and fires when user ends trip in CurrentTrip Modal view 
TripExtendedMessage added to CurrentTrip, fires when user adds more time to their trip 
TripNotCompleteMessage added to CurrentTrip, fires when 30 seconds runs out in Alert Modal 

## How has this been tested?
Tested user paths for start trip, adding time, ending trip, not complete trip messaging 
- [X] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman
- [X] Mobile

### CODE REVIEW

File: src/Components/TripExtendedMessage/TripExtendedMessage.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+export function TripExtendedMessage(user, extension, contact) {
+
+  const sendSms = () => {
+
+    let smsObj = {
+      mobile_number: `${contact.phone}`,
+      message: `${user.firstName} has extended their trip by ${extension} minutes. Please be on the look out for the trip completed confirmation`,
```
Comment (pcmueller) [inline src/Components/TripExtendedMessage/TripExtendedMessage.js:8]: Nice job incorporating the time extension amount dynamically!

File: src/Components/TripExtendedMessage/TripExtendedMessage.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        accepts: "application/json"
+      },
+      body: JSON.stringify(smsObj)
+    })
+        .then(result => result.text())
+        .then(resp => console.log(resp))
+  }
+
+  return(
+      sendSms()
```
Comment (pcmueller) [inline src/Components/TripExtendedMessage/TripExtendedMessage.js:24]: So is this POST request what's actually sending the text message?

File: src/Components/TripStartMessage/TripStartMessage.js [hunk 3]
```diff
@@ -1,30 +1,28 @@
...
-          'content-type': 'application/json',
-          accepts: "application/json"
-        },
-        body: JSON.stringify(smsObj)
-      })
-          .then(result => result.text())
-          .then(resp => console.log(resp))
+    let smsObj = {
+      mobile_number: `${contact.phone}`,
+      message: `${userName.firstName} has started a trip from ${textInformation.startPoint} to ${textInformation.endPoint}. ${userName.firstName} is traveling by ${textInformation.travelMode} with an ETA of ${textInformation.eta} minutes. Please be on the look out for a follow up message from ${userName.firstName}`,
```
Comment (pcmueller) [inline src/Components/TripStartMessage/TripStartMessage.js:8]: Now that's what I call interpolation!

**Review by pcmueller — APPROVED** [review]
Thanks for figuring all of this out, nice work!

**Review by carolineshowscode — APPROVED** [review]
Thanks for posting the demo in our Slack group. The alert messages look great

**Review by foymikek — APPROVED** [review]
Nice job on the SMS feature, super solid!

**Pull request merged**